agents/gce/fence_gce.py

```diff
@@ -148,17 +148,20 @@ def translate_status(instance_status):
 
 def get_nodes_list(conn, options):
 	result = {}
+	action = options["--action"] if "--action" in options else ""
 	plug = options["--plug"] if "--plug" in options else ""
 	zones = options["--zone"] if "--zone" in options else ""
 	filter = "name="+plug if plug != "" else ""
+        max_results = 1 if action == "monitor" else 500
```

Comment:
Great. You should unindent this line though, so it follows the indentation of the lines above.